.github/workflows/prepare_release.yml

```diff
@@ -74,6 +79,7 @@ jobs:
           commit-message: "Release ${{ inputs.version }}"
           branch: release-prep
           delete-branch: true
+          sign-commits: ${{ inputs.sign-commits }}
```

Comment:
Q: Given the `github_pat` secret is required, and passed to the `token`, can't we just hardcode this parameter to `true`?